packages/panzer/adapters-stk/test/evaluator_tests/gather_solution.cpp

```diff
@@ -89,8 +89,8 @@
 #include "user_app_EquationSetFactory.hpp"
 
 typedef double ScalarT;
-typedef int LocalOrdinalT;
-typedef int GlobalOrdinalT;
+using LocalOrdinalT = Tpetra::Map<>::local_ordinal_type;
+using GlobalOrdinalT = Tpetra::Map<>::global_ordinal_type;
```

Comment:
To be consistent with the rest of panzer, these should be panzer::LocalOrdinal and panzer::GlobalOrdinal. In deprecated code we need to make sure long long is preferred GO type.

Reply:
The idea is that Panzer prefers `long long`, regardless of what Tpetra enables.  If Tpetra doesn't enable `long long`, then Panzer won't build, and that's OK.  Is that right?

Reply:
close. If long long is enabled as tpetra GO (when multiple types enabled) then panzer will always prefer long long over any other type. The apps expect this for runs on big machines. But panzer will use any GO if long long is not enabled and falls back on tpetra default. The issue is that the deprecated tpetra path preferred int over long long if multiple defaults were enabled, so panzer defined panzer::GlobalOrdinal to override. In the new tpetra code path long long is always the default if enabled so we can simplify panzer once the deprecated path is removed.